types/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+      "lib": ["es2015"],
+      "strict": true,
+      "baseUrl": ".",
+      "paths": {
+          "unist-util-inspect": ["index.d.ts"]
+      }
+  },
```

This comma is causing an invalid json error

```suggestion
  }
```